### PR TITLE
Set VersionSuffix to empty instead of use prefix

### PR DESCRIPTION
**Pull request by wli3**

### CODE REVIEW

**Comment by vijayrkn** [comment]
Fix for https://github.com/aspnet/Scaffolding/issues/932

**Comment by vivmishra** [comment]
Approved for 2.2.1

**Comment by seancpeters** [comment]
@wli3 - could you give a brief description of the intended effects of this change? Thanks!

File: version.props [hunk 1]
```diff
@@ -38,12 +38,10 @@
 
     <!-- Run the build with /p:IsFinalBuild=true to produce the product with 'final' branding and versioning -->
     <IsFinalBuild Condition=" '$(IsFinalBuild)' == '' ">false</IsFinalBuild>
-    <IncludePreReleaseLabelInPackageVersion>true</IncludePreReleaseLabelInPackageVersion>
-    <IncludePreReleaseLabelInPackageVersion Condition=" '$(IsFinalBuild)' == 'true' AND ('$(PreReleaseLabel)' == 'servicing' OR '$(PreReleaseLabel)' == 'rtm')">false</IncludePreReleaseLabelInPackageVersion>
+    <IncludePreReleaseLabelInVersion>true</IncludePreReleaseLabelInVersion>
```
Comment (natemcmaster) [inline version.props]: Can you keep this variable as-is? It's used in other places.

File: version.props [hunk 2]
```diff
@@ -38,12 +38,10 @@
...
+    <IncludePreReleaseLabelInVersion Condition=" '$(IsFinalBuild)' == 'true' AND ('$(PreReleaseLabel)' == 'servicing' OR '$(PreReleaseLabel)' == 'rtm')">false</IncludePreReleaseLabelInVersion>
 
-    <!-- The version in files -->
-    <PackageVersion>$(VersionPrefix)</PackageVersion>
-    <PackageVersion Condition=" '$(IncludePreReleaseLabelInPackageVersion)' == 'true' ">$(VersionPrefix)-$(VersionSuffix)</PackageVersion>
+    <VersionSuffix Condition=" '$(IncludePreReleaseLabelInVersion)' == 'false' "></VersionSuffix>
```
Comment (natemcmaster) [inline version.props]: I think a simpler fix would be to leave the file as is and just add 
```xml
<Version>$(PackageVersion)</Version>
```
VersionPrefix/VersionSuffix end up in other places, like informational version attributes, which is useful for figuring out the exact build number used to produce our packages and binaries.

**Comment by vijayrkn** [comment]
Based on the conversation with Nate, the simplest change would be to add this to the CodeGenerator csproj so that the change is isolated to just this scenario.

`<Version>$(PackageVersion)</Version>`

**Comment by wli3** [comment]
@natemcmaster I reset my change and only add the property change to csporj. I tested with `build.cmd /p:IsFinalBuild=true` and then global tool install locally to ensure no repro

**Comment by vijayrkn** [comment]
Can we merge this change and get the PR ready to update the submodule?

**Comment by seancpeters** [comment]
I want to do one more quick test. After that, if no one else has concerns, I’ll merge it and get the submodules PR setup.

**Pull request merged**